docs/tutorials/kev-dev-with-skaffold.md

```diff
@@ -0,0 +1,97 @@
+
+## ⚬ Initialise Kev project with Skaffold support
+
+In order to take advantage of Skaffold, you must prepare your project accordingly. Provided you haven't initialised Kev application yet it's as simple as running the following command:
+
+> Run Kev in dev mode with Skaffold dev loop enabled.
+```sh
+# 1) Starts watch loop and automatically re-renders K8s manifests
+#    for specified environments.
+# 2) Observes application source code and K8s manifests changes
+#    and triggers build/push/deploy.
+
+kev init --skaffold -e dev -e ...
+```
+This command prepares your application and bootstraps a new Skaffold config (_skaffold.yaml_) if it doesn't already exist. Alternatively, it'll add environment & helper profiles to already existing Skaffold config automatically. The profiles added by Kev can be used to control which application Kubernetes manifests should be deployed and to which K8s cluster, be it local or remote. They should also come handy when defining steps in CI/CD pipelines.
+
+## ⚬ Retrofit Skaffold support in existing Kev project
+
+If Kev project has been previously initialised without Skaffold support, the easiest way forward to adopt Skaffold is to remove _kev.yaml_ file and initialize the project again. Be mindful that names of all the environments you want to track must be specified - _Kev_ `init` won't automatically discover existing environment override files!
+
+Alternatively, use `skaffold init` to bootstrap _skaffold.yaml_ and tell Kev about the fact by adding the following line in _kev.yaml_ file:
+
+> Initialize Skaffold in your project
+```sh
+# It'll interactively build skaffold.yaml based on user choices.
+# The configuration will be saved in skaffold.yaml file.
+
+skaffold init
+```
+
+> Add the following line in kev.yaml file.
+```yaml
+compose:
+  - ...
+environments:
+  dev: ...
+skaffold: skaffold.yaml # <= tell Kev that skaffold is now initialised
+```
+
+## ⚬ Kev + Skaffold
+
+At this point all you need to do to take advantage of Skaffold integration is to start _Kev_ in [development](cli/kev_dev.md) mode with Skaffold hook enabled:
+
+> Start kev in development with Skaffold integration activated
+```sh
+kev dev --skaffold
+```
+
+The command will start two watch loops,
+1) one responsible for reconciling changes in your Docker Compose project source and environment override files to produce up-to-date Kubernetes manifests for each of specified environments (or default `dev` environment if none provided),
+2) and second loop responsible for watching changes in your project source code and deployment manifests.
+
+Every change to Docker Compose project will produce updated set of K8s manifests for your app, which in turn will inform Skaffold to trigger another Build/Push/Deploy iteration. This will deploy fresh set of manifests to the target Kuberentes cluster.
```

Comment:
`Every change to Docker Compose project...` -> `Every change made to the Docker Compose project...`